### PR TITLE
Use danger

### PR DESCRIPTION
**Pull request by Bugagazavr**

### CODE REVIEW

**Comment by Bugagazavr** [comment]
@dblock can you add token, please?

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,10 @@ cache: bundler
 
 matrix:
   include:
-    - rvm: 2.3.0
```
Comment (dblock) [inline .travis.yml]: We should probably keep 2.3.0 around too, no? No big deal.

File: Gemfile [hunk 2]
```diff
@@ -5,6 +5,10 @@ gemspec
 
 gem 'grape-swagger', github: 'ruby-grape/grape-swagger'
 
+unless RUBY_PLATFORM == 'java'
```
Comment (dblock) [inline Gemfile]: RuboCop would complain. I'd make sure Gemfile is part of .rubocop.yml and `rubocop -a` this one.

**Comment by dblock** [comment]
@Bugagazavr Added. And it flagged two CHANGELOG format problems. You can't even imagine how happy I am we're fixing those! ;)

**Pull request merged**